server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -328,8 +328,8 @@ synchronized void setManagerState(final ManagerState newState) {
         break;
       case HAVE_LOCK:
         if (isUpgrading()) {
+          upgradeCoordinator.continueUpgrade();
           upgradeCoordinator.preUpgradeValidation();
```

Comment:
I don't think the preUpgradeValidation needs to run inside the manager at all. Its function is superseded by these changes that initiates the upgrade process before the manager is run. Once the upgrade has started, it doesn't have any value.

Reply:
Removed in 6a21a3f